src/args.rs

```diff
@@ -19,6 +19,9 @@ pub mod args {
         /// The output scale (1 is the original size)
         #[arg(short, long, default_value = "4")]
         pub scale: u32,
+        // Enstablish how much wide is the output images, in columns. Overrides `scale`
```

Comment:
Can you make this comment a document comment(`///`) instead of a normal comment, because the `clap` will be use it in the help message that appears when you run `aarty --help`